src/components/Clusters/ClusterDetail.tsx

```diff
@@ -1,5 +1,140 @@
+import { Button, Card, CardSection, Group, Text } from '@mantine/core';
+import { useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { RiDeleteBin5Line, RiSettings4Line, RiToggleFill, RiToggleLine } from 'react-icons/ri';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { useAuth } from '@/hooks/useAuth';
+import { useClusterDetails, useJobsOnCluster } from '@/hooks/useClusters';
+import { disableModal, enableModal } from '@/utils/modals';
+import { pluralize } from '@/utils/string';
+
+import ClusterConfigCard from './ClusterConfigCard';
```

Comment:
Try to use the @/ syntax for all imports.

Reply:
OK 👍🏻, are there any additional benefits to using this approach aside from improving readability.